apps/masterbots.ai/components/routes/browse/browse-thread-blog.tsx

```
@@ -0,0 +1,221 @@
+            }`}
+          >
+            {/* User Questions as Headers */}
+            {isUser && index === 0 && (
+              <header>
+                <h1 className="text-3xl font-bold ">
+                  {message.content}
+                </h1>
+              </header>
+            )}
+
+            {isUser &&  index != 0 && (
```

Biome (1.9.4)

[error] Use !== instead of !=. != is only allowed when comparing against `null`

!= is only allowed when comparing against null

Using != may be unsafe if you are relying on type coercion
Unsafe fix: Use !==


(lint/suspicious/noDoubleEquals)